src/action/common/place_nix_configuration.rs

```diff
@@ -41,9 +44,12 @@ impl PlaceNixConfiguration {
             extra_conf = extra_conf.join("\n"),
             version = env!("CARGO_PKG_VERSION"),
         );
-        let create_directory =
-            CreateDirectory::plan(NIX_CONF_FOLDER, None, None, 0o0755, force).await?;
-        let create_file = CreateFile::plan(NIX_CONF, None, None, 0o0664, buf, force).await?;
+        let create_directory = CreateDirectory::plan(NIX_CONF_FOLDER, None, None, 0o0755, force)
+            .await
+            .map_err(|e| ActionError::Child("create-directory", Box::new(e)))?;
+        let create_file = CreateFile::plan(NIX_CONF, None, None, 0o0664, buf, force)
+            .await
+            .map_err(|e| ActionError::Child("create-file", Box::new(e)))?;
```

Comment:
Should this either be using `CreateDirectory`/`CreateFile`'s `typetag` method, or be snake case (as elsewhere)?

Reply:
It most certainly should be, thanks!